src/configs/chains/kusama-chains.ts

```diff
@@ -139,4 +139,20 @@ export const kusamaChains = {
     paraChainId: 2118,
     ss58Prefix: 42,
   },
+  robonomics: {
+    id: "robonomics",
+    display: "Robonomics",
+    type: "substrate",
+    icon: "https://resources.acala.network/_next/image?url=%2Fnetworks%2Frobonomics.png&w=96&q=75",
+    paraChainId: 2048,
+    ss58Prefix: 32,
+  },
+  tinkernet: {
+    id: "tinkernet",
+    display: "Tinkernet",
+    type: "substrate",
+    icon: "",
```

Comment:
It's better to provide an icon source, you may add the icon into resouces.acala.network through this repo: https://github.com/open-web3-stack/token-resources